src/ngx_http_lua_common.h

```diff
@@ -548,6 +548,9 @@ typedef struct ngx_http_lua_ctx_s {
     ngx_chain_t             *busy_bufs;
     ngx_chain_t             *free_recv_bufs;
 
+    ngx_chain_t             *filter_in_bufs;  /* for the body filter */
+    ngx_chain_t             *filter_busy_bufs;  /* for the body filter */
```

Comment:
can we use union here to save memory?
like:
```
union {
    ngx_chain_t *filter_busy_bufs;
    ngx_chain_t *busy_bufs;
};
```

Maybe `filter_in_bufs` can union with `free_recv_bufs`?